src/main/java/com/google/sps/servlets/GoogleBooksServlet.java

```diff
@@ -66,21 +65,50 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             params += "&startIndex=" + startIndex;
         }
 
-        //get .properties file that stores the api key
-        Properties prop = new Properties();
-        InputStream input = getClass().getResourceAsStream("/api.properties");
-        prop.load(input);
+        HttpURLConnection connection = getConnection(params);
+        connection.setRequestMethod("GET");
+        connection.setRequestProperty("key", getApiKey());
+        connection.setRequestProperty("User-Agent", "Mozilla/4.76");
```

Comment:
Could you explain what this User-Agent property does?

Reply:
It lets the server know what type of application is being used. In this case, I set it to use the Mozilla web browser.